libnavui-voice/src/test/java/com/mapbox/navigation/ui/voice/api/PreOreoAudioFocusDelegateTest.kt

```diff
@@ -32,6 +37,118 @@ class PreOreoAudioFocusDelegateTest {
         }
     }
 
+    @Test
+    fun `pre oreo audio delegate returns true when audio focus is granted`() {
+        val mockedAudioManager = mockk<AudioManager>(relaxed = true)
+        val mockedVoiceInstructionsPlayerOptions: VoiceInstructionsPlayerOptions = mockk()
+        every {
+            mockedVoiceInstructionsPlayerOptions.focusGain
+        } returns AudioManager.AUDIOFOCUS_GAIN_TRANSIENT_MAY_DUCK
+        every {
+            mockedVoiceInstructionsPlayerOptions.streamType
+        } returns AudioManager.STREAM_MUSIC
+        every {
+            mockedAudioManager.requestAudioFocus(
+                null,
+                AudioManager.STREAM_MUSIC,
+                AudioManager.AUDIOFOCUS_GAIN_TRANSIENT_MAY_DUCK
+            )
+        } returns AudioManager.AUDIOFOCUS_REQUEST_GRANTED
+
+        val preOreoAudioFocusDelegate = PreOreoAudioFocusDelegate(
+            mockedAudioManager,
+            mockedVoiceInstructionsPlayerOptions
+        )
+
+        Assert.assertEquals(
```

Comment:
```suggestion
        assertEquals(
```

The same thing applies to other tests 👇